components/gallery/GalleryItem.vue

```diff
@@ -98,10 +98,15 @@ const CarouselTypeVisited = defineAsyncComponent(
 </script>
 
 <style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
```

Comment:
import variable just for one color? ☹️ 
couldn't this be done in global stylesheet?